source/framefmt.c

```diff
@@ -213,6 +214,65 @@ static inline int serde_nonframed(
     return AWS_ERROR_SUCCESS;
 }
 
+bool aws_cryptosdk_frame_is_valid(const struct aws_cryptosdk_frame *const frame) {
+    if (frame == NULL) {
+        return false;
+    }
+
+    bool frame_type_seq_valid = aws_cryptosdk_frame_has_valid_type(frame);
```

Comment:
This doesn't check the sequence number, so should not be `seq_valid`

Reply:
Thanks for the catch.